CodeTheDream/ride-share-app
# updated acceptride api call to scheduleride

## Description

Update API call from 'accepted' to 'scheduled' and changed the function name in component "RequestedRidesDetails" accordingly.

## Background Information
There was a confusion over scheduled rides and accepted rides, and this PR makes things clear and consistent. (there are now scheduleride and scheduledride functions.

## Screenshots

## Checklist:
- [ ] Screenshots attached (if dealing with UI)

## Testing Steps for reviewer:
- Step 1
Check "RequestedRidesDetails.js" and "api.js".

*NOTE: We need backend people to change the names on their end in order to complete this change and make it work.
